core/security.py

```diff
@@ -20,9 +16,3 @@ async def get_api_key(api_key_header: str = Security(api_key_header)):
             status_code=403,
             detail="Could not validate credentials",
         )
-
-
-def generate_custom_id():
```

Comment:
Removed as its only used in 1 function, and we can use the default python `uuid` library